### PR TITLE
ServiceNow Mirror Updates

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
- Added new integration parameters:
    - Incident Mirroring Direction
    - File Entry Tag
    - Work Note Entry Tag
    - Comment Entry Tag

- Fixed an issue where the date format was incorrect
- Updated the classifier with the new integration parameters.

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Documentation

### CODE REVIEW

File: Packs/ServiceNow/Classifiers/classifier-mapper-incoming-ServiceNow.json [hunk 1]
```diff
@@ -559,8 +559,44 @@
                     "simple": "subcategory"
                 },
                 "Ticket Closed Date": {
-                    "complex": null,
-                    "simple": "closed_at"
+                    "complex": {
```
Comment (Itay4) [inline Packs/ServiceNow/Classifiers/classifier-mapper-incoming-ServiceNow.json:562]: shouldnt we do the same for due date?

File: Packs/ServiceNow/Classifiers/classifier-mapper-incoming-ServiceNow.json [hunk 2]
```diff
@@ -609,10 +654,10 @@
                     "complex": null,
                     "simple": ".='ServiceNow v2_instance_1'"
                 },
-                "dbotMirrorTags": {
```
Comment (Itay4) [inline Packs/ServiceNow/Classifiers/classifier-mapper-incoming-ServiceNow.json:612]: please fix lint

File: Packs/ServiceNow/Classifiers/classifier-mapper-incoming-ServiceNow.json [hunk 3]
```diff
@@ -609,10 +654,10 @@
                     "complex": null,
                     "simple": ".='ServiceNow v2_instance_1'"
```
Comment (Itay4) [inline Packs/ServiceNow/Classifiers/classifier-mapper-incoming-ServiceNow.json:655]:
```suggestion
                    "simple": "mirror_instance"
```

File: Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml [hunk 4]
```diff
@@ -87,6 +87,35 @@ configuration:
   name: close_ticket
   required: false
   type: 8
+- defaultvalue: None
```
Comment (Itay4) [inline Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml:90]: lets change a bit the params order:
below fetch params place the mirroring params, where the new ones are the first and then the Close params 
below the mirror params place the insecure and proxy

File: Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml [hunk 5]
```diff
@@ -87,6 +87,35 @@ configuration:
...
+  display: Incident Mirroring Direction
```
Comment (Itay4) [inline Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml:91]: please add tooltips to all new params with short explanation what each means

File: Packs/ServiceNow/Integrations/ServiceNowv2/test_data/response_constants.py [hunk 6]
```diff
@@ -40,7 +40,7 @@
     'group_list': '', 'work_end': '', 'caller_id': '',
     'reopened_time': '', 'resolved_at': '', 'approval_set': '', 'subcategory': '', 'work_notes': '',
     'short_description': 'Trouble getting to Oregon mail server', 'close_code': '', 'correlation_display': '',
-    'delivery_task': '', 'work_start': '', 'assignment_group': {'link': 'demisto.com', 'value': 'test'},
+    'delivery_task': '', 'work_start': '', 'assignment_group': '',
```
Comment (Itay4) [inline Packs/ServiceNow/Integrations/ServiceNowv2/test_data/response_constants.py:43]: please make sure we test `assignment_group`

**Comment by Itay4** [comment]
please enhance the readme with the new params and edit the mirror setup accordingly

**Comment by Itay4** [comment]
this doesnt look well, need to move the image to be below the text (also it seems like we need to update the image, i dont think we have a parameter `Entry Tags`):
image

**Comment by Shellyber** [comment]
@Itay4 Done.

**Review by Itay4 — APPROVED** [review]
nice

**Comment by kirbles19** [comment]
@Shellyber Done.

**Pull request merged**